<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElecteurInterfaceClass</class>
 <widget class="QDialog" name="ElecteurInterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ElecteurInterface</string>
  </property>
  <widget class="QLabel" name="label_rue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom de rue :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_valideElecteur">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_nom">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dateNaissance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Date de naissance :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nas">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;table border=&quot;0&quot; style=&quot;-qt-table-type: root; margin-top:4px; margin-bottom:4px; margin-left:4px; margin-right:4px;&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot;border: none;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Numero d'assurance&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;social :&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nCivic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero civic :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_titre">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>AJout d'un Electeur</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_ville">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ville :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_prenom">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Prenom :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_codePostal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Code postal :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_province">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Province :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nom">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_prenom">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximumDate">
    <date>
     <year>7937</year>
     <month>12</month>
     <day>31</day>
    </date>
   </property>
   <property name="minimumDate">
    <date>
     <year>1770</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>dd-MM-yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nas">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nCivic">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_rue">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ville">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_codePostal">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_province">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>360</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Annuler">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_valideElecteur</sender>
   <signal>clicked()</signal>
   <receiver>ElecteurInterfaceClass</receiver>
   <slot>validerElecteur()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Annuler</sender>
   <signal>clicked()</signal>
   <receiver>ElecteurInterfaceClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>validerElecteur()</slot>
 </slots>
</ui>
